messages/ar.json

```diff
@@ -44,6 +44,26 @@
   "admin/b2b-organizations.bulk-import.import.fileExtensionError": "Sorry, the selected file format is not supported.",
   "admin/b2b-organizations.bulk-import.import.fileSizeError": "Sorry, the file size is too big.",
   "admin/b2b-organizations.bulk-import.import.importErrorTryAgain": "Try Again",
+  "admin/b2b-organizations.bulk-import.import.reportInformationTitle": "{fullPercentage}% Import Success",
+  "admin/b2b-organizations.bulk-import.import.reportInformationDescription": "File {fileName} uploaded by {userName} on {uploadDate}",
+  "admin/b2b-organizations.bulk-import.import.reportInformationListTotal": "Total",
+  "admin/b2b-organizations.bulk-import.import.reportInformationImportSuccessFully": "Imported Successfully",
+  "admin/b2b-organizations.bulk-import.import.reportInformationImportFailed": "Failed to Import",
+  "admin/b2b-organizations.bulk-import.import.reportScreenTitle": "We found {errorCount} errors in this file.",
+  "admin/b2b-organizations.bulk-import.import.reportScreenDescription": "File {fileName} uploaded by {userName} on {uploadedDate}",
+  "admin/b2b-organizations.bulk-import.import.reportScreenLabel": "Total errors",
+  "admin/b2b-organizations.bulk-import.import.reportScreenSuccessCount": "{successCount} Buyer Organizations",
+  "admin/b2b-organizations.bulk-import.import.reportInformationInitialUpload": "To address the errors, download the <a>Reviewed.XLSX</a>. It shows the status of each row — green for success and red for failure. Errors and their details are provided, where applicable, in the last errors column of each tab.",
```

Comment:
nit: I suppose the name of the file won't be "Reviewed.XLSX" always (we probably going to use the original file name + "reviewed" postfix or something like that. Maybe change it to `<a>reviewed xlsx</a>` ?

Reply:
Yes, I believe this was supposed to include a file name variable, right @ArthurTriis1 ? Something like: `<a>{fileName}</a>`.

Reply:
Yes, will be better to use the fileName, i will change